internal/controller/csr_controller.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
As discussed with the team the CSI operator should not be responsible for generation of certificates. ODF operators can generate these certificates.